ksql-rest-app/pom.xml

```diff
@@ -113,6 +113,20 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.kjetland</groupId>
+            <artifactId>mbknor-jackson-jsonschema_2.12</artifactId>
```

Comment:
This is the JSON schema generation library. I played with a few, and this was the only one that correctly handles interfaces and Jackson annotations. It uses the MIT license.